code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -113,6 +113,18 @@
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	slot_flags = SLOT_BELT
 	structure_damage_factor = STRUCTURE_DAMAGE_BLADE
+	var/backstab_damage = 8
+
+/obj/item/weapon/tool/knife/resolve_attackby(atom/target, mob/user)
+	..()
+	if(iscarbon(target) && get_turf(target) == get_step(user, user.dir) && target.stat!=2 && user.dir == target.dir) //Detect that we're attacking a carbon target, that we're behind it as well.
```

Comment:
Use early returns and break this down.

Reply:
Changed. Tested as well real quick, still works.